<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberConnectDialog</class>
 <widget class="QDialog" name="MemberConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign up</string>
  </property>
  <layout class="QVBoxLayout" name="_2" stretch="1,0,0,0">
   <property name="spacing">
    <number>4</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a name=&quot;result_box&quot;/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Sign in to access your series.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="2,1,1">
     <property name="topMargin">
      <number>12</number>
     </property>
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="1" column="0">
      <widget class="QLineEdit" name="loginLe"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="passwdLbl">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="loginLbl">
       <property name="text">
        <string>BetaSeries login</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="passwdLe">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="forgetLbl">
       <property name="text">
        <string>&lt;a href=&quot;forget&quot; style=&quot;text-decoration:none&quot;&gt;Forgot?&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="errorLbl">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MemberConnectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MemberConnectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
